signac/errors.py

```diff
@@ -80,6 +73,7 @@ class KeyTypeError(TypeError):
     "IncompatibleSchemaVersion",
     "InvalidKeyError",
     "JobsCorruptedError",
+    "KeyTypeError",
```

Comment:
You may want to merge `master` up into your PR -- I also did this same thing in another PR.

Reply:
Hmm I just did that earlier today and this didn't get merged. I just double-checked the diff from that merge and this didn't change, I'm not sure why. We can double check when I open the synced_collections->master PR.